src/search_renderer.cpp

```diff
@@ -95,7 +95,7 @@ std::string SearchRenderer::getHtml()
     result.set("resultContentId", mp_nameMapper->getNameForId(s.str()));
     std::shared_ptr<zim::Archive> archive;
     try {
-       result.set("bookTitle", mp_library->getBookById(s.str()).getTitle());
+       result.set("bookTitle", archive ? mp_library->getBookById(s.str()).getTitle() : kainjow::mustache::data(false));
```

Comment:
This fix simply undoes the functionality introduced by #705. If this fix is correct (but it obviously isn't) it can be simplified to

```
result.set("bookTitle", kainjow::mustache::data(false));
```
since `archive` is always `nullptr`¹. Then, given how the `bookTitle` parameter is used during mustache template rendering, you could instead have deleted the entire `try/catch` statement, with the same effect.

Yet, once again, that's not how the crash should be fixed.

---
¹ The `archive` variable was introduced in the initial version of PR #705 and I overlooked that it became unused after requested changes; please remove that pointless variable